### PR TITLE
Add id attribute to username, password and signin button

### PR DESCRIPTION
**Pull request by manueltanzi-okta**
🌱 Add id attribute to username, password and signin button

Add id attribute to username, password and signin button and bump courage version to 2.6.0

Resolves: [OKTA-125825](https://oktainc.atlassian.net/browse/OKTA-125825)

@hor-kanchan-okta @alexeystadnikov-okta @mauriciocastillosilva-okta

### CODE REVIEW

**Comment by manueltanzi-okta** [comment]
@mauriciocastillosilva-okta bump review thank you :)

File: src/views/primary-auth/PrimaryAuthForm.js [hunk 1]
```diff
@@ -87,6 +89,7 @@ define([
         placeholder: Okta.loc('primaryauth.password.placeholder', 'login'),
         name: 'password',
         input: TextBox,
+        inputId: 'pass-signin',
```
Comment (mauriciocastillosilva-okta) [inline src/views/primary-auth/PrimaryAuthForm.js]: nit: why not ```password-signin```? I think it will be a more descriptive id for this input

Comment (hor-kanchan-okta) [inline src/views/primary-auth/PrimaryAuthForm.js]: We try to keep the id same as the old signin page.

**Comment by rchild-okta** [comment]
Why do we need to add these id's? Is the ops monitoring not able to use other types of selectors?

If we have to add ids, they should be more specific - remember that any id we use could collide with any name a developer would choose for their own app. Something like "okta-signin-widget-user", etc (might be enough to just prefix with okta).

File: src/views/primary-auth/PrimaryAuthForm.js (hunk 1 above)
Comment (rchild-okta) [inline src/views/primary-auth/PrimaryAuthForm.js]: If we need consistency, we should change the fields in the old loginform to whatever we end up with here.

**Comment by hor-kanchan-okta** [comment]
From what @garethpaterson-okta told me, the Ops monitoring only use id. It is possible to use other type but it will be inconsistent with the rest of the monitoring script.

For the id, i was trying to keep the same id. So the monitoring script will work for both new and old signin flow. But @rchild-okta has a good point that the id may be collided.
So we have to update the id. Add a prefix.
If necessary, we can update the old signin page ids.
@garethpaterson-okta , will different work for you?

**Comment by hor-kanchan-okta** [comment]
@garethpaterson-okta is ok with different ids. 
@manueltanzi-okta can you change the id to okta-signin-username, okta-signin-password, okta-signin-submit
@rchild-okta I suggest adding the id which will make Ops life easier.

**Pull request merged**